gym-service/src/main/java/org/example/externaldto/TrainingEventDto.java

```diff
@@ -0,0 +1,43 @@
+package org.example.externaldto;
```

Comment:
So, i think you're updating both services to have absolutely same classes in absolutely same packages. Such duplications and code updates only for purpose of being in same packages should be avoided. Rather than this I recommend 2 approaches:

- extract library to contain all the dtos and reuse it in services
- write/read simple string json, which you can create with `ObjectMapper` and use `writeValueAsString()`/`readValue()` without additional converter